src/lib/index.ts

```diff
@@ -0,0 +1 @@
+export {CalculateEnergyConsumption} from './power-to-energy';
```

Comment:
`CalculateEnergyConsumption` does not exist in this PR - you should update to valid function name.